conans/test/integration/command/test_package_test.py

```diff
@@ -128,46 +130,6 @@ def test_build_folder_handling(self):
                                                     "build_folder")))
         self.assertFalse(os.path.exists(default_build_dir))
 
-    def test_check_version(self):
```

Comment:
how is this related?

Reply:
I removed it because it uses the old cpp_info attributes in the test: 
```
            class Dep(ConanFile):
                def package_info(self):
                    self.cpp_info.name = "MyDep"
```